gcc/rust/ast/rust-expr.h

```diff
@@ -2139,6 +2168,8 @@ class CallExpr : public ExprWithoutBlock
     return *function;
   }
 
+  std::unique_ptr<Expr> &get_function_expr_ptr () { return function; }
```

Comment:
Unless you need to reseat that pointer a returning an `Expr &` alone should be sufficient.

Reply:
I need to reseat it for example for the `?` operator desugar. at least with the current design